lisa/analysis/eas.py

```diff
@@ -0,0 +1,72 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2020, ARM Limited and contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pandas as pd
+import numpy as np
+
+from lisa.analysis.base import TraceAnalysisBase
+from lisa.trace import requires_events
+from lisa.datautils import df_delta
+
+class EASAnalysis(TraceAnalysisBase):
+    """
+	Energy Aware Scheduler (EAS) specific analysis.
```

Comment:
there seems to be a mix of space and tabs for indentation, all tabs should be replaced by 4 spaces